### DISCUSSION TITLE
Duplicate Book Files

### DISCUSSION THREAD

**Discussion by kilmerc** [openaudible/openaudible#898] [Q&A] [answered]
Most all my books were in M4A Format, then in the preferences, I noticed M4B format was recommended so I changed the setting. OA then automatically converted a lot of my M4A files to M4B but left the books in the M4A format as well so now I have duplicate copies of a lot, but not all my books. Is this intended behavior and is there a way for me to clear out the old files without having to check each file one by one to determine if it was converted or not? I have over 400 books and would prefer to not have to do that...
<img width="128" alt="image" src="https://user-images.githubusercontent.com/20101738/157175382-170b1430-598f-4e8e-9387-1162ee9fb442.png">

**Comment by openaudible** [comment, marked as answer]
There isn't anything built-in to detect duplicates. If all of your books are from audible, you can make a backup of everything and delete (or move) the m4a files and re-encode them. Running "Controls: Refresh Files" (before and after) can also find books can help. 
But if you've added your own m4a files you'll just have to be careful about going through them.
I don't think our software changes m4a to m4b automatically-- but it will start using m4b format for new conversions once you make that change. 
Generally the m4a and m4b formats are the same, so no reason to keep both formats.
Good luck.

**Reply by kilmerc** [reply]
Thanks for the response! Love the software.
